doc/src/man/linkcheckerrc.rst

```diff
@@ -503,6 +503,14 @@ WordParser
 Parse Word files for URLs to check. Needs the :pypi:`pywin32` Python
 extension installed.
 
+MarkdownCheck
+^^^^^^^^^^^^^
+
+Parse Markdown files for URLs to check.
+
+**filename_re=**\ *REGEX*
+    Regular expression matching the names of Markdown files.
```

Comment:
It would be nice to document the default value, or at least provide an example.

Reply:
I started with that (there are examples in other sections of the page), but then realised that none of the plugins have documented examples. Probably worth adding for all.

Reply:
A small improvement today is better than a large improvement next week/month/year/never.  ;)

Reply:
Raised #506 to record.